include/xnvme_be_macos_driverkit.h

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: Samsung Electronics Co., Ltd
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifndef __INTERNAL_XNVME_BE_MACOS_DRIVERKIT_H
+#define __INTERNAL_XNVME_BE_MACOS_DRIVERKIT_H
+#include <IOKit/IOKitLib.h>
+#include <xnvme_be.h>
+
+#include "MacVFNShared.h"
+
+/**
+ * Internal representation of XNVME_BE_MACOS_DRIVERKIT state
+ *
+ * NOTE: When changing this struct, ensure compatibility with 'struct xnvme_be_cbi_state'
+ */
+struct buffer;
```

Comment:
Move the buffer up before the doc-string, since the docstring should be before the thing that it documents.

Reply:
Done